test/remote/gateways/remote_litle_test.rb

```diff
@@ -498,21 +503,20 @@ def test_void_with_echeck
     })
     assert sale = @gateway.purchase(2004, @check, options)
 
-    assert void = @gateway.void(sale.authorization)
+    assert void = @gateway.void(sale.authorization, options)
     assert_success void
     assert_equal 'Approved', void.message
   end
 
   def test_void_authorization
     assert auth = @gateway.authorize(10010, @credit_card1, @options)
-
-    assert void = @gateway.void(auth.authorization)
+    assert void = @gateway.void(auth.authorization, @options)
     assert_success void
     assert_equal 'Approved', void.message
   end
 
   def test_unsuccessful_void
-    assert void = @gateway.void('123456789012345360;authorization;100')
+    assert void = @gateway.void('123456789012345360;authorization;1000', @options)
```

Comment:
:speech_balloon:
Maybe you can change this for:
```suggestion
    authorization = '123456789012345360;authorization;1000'
    assert void = @gateway.void(authorization, @options)
```

or

create first an authorize and get the authorization
```suggestion
    assert authorize = @gateway.authorize(10010, @credit_card1, @options)@options)
    assert void = @gateway.void(authorize.authorization, @options)
```

Reply:
sandbox behavior doesn't allow to use a previous response as a parameter to later request

Reply:
ohhh, just I realized that doing that would be a successful void. :grin:, then  the test is failing because it should be `assert_success` instead `assert_failure` and besides there is another test called `test_void_unsuccessful ` in the line 597